### ISSUE TITLE
You could use issues instead of note cards in your projects boards.

### ISSUE THREAD

**Issue by moeenio** [MasterOfTheTiger/reimagine-scratch#3]

**Comment by MasterOfTheTiger** [comment]
Yeah, you are probably right, but there are many small ideas I wanted to implement quickly, and I haven't worked on this for a while.

The biggest thing I need to do right now is to find out how to use JS to modify the new design (which I haven't figured out yet).

**Comment by moeenio** [comment]
> Yeah, you are probably right, but there are many small ideas I wanted to implement quickly, and I haven't worked on this for a while.
> 
> The biggest thing I need to do right now is to find out how to use JS to modify the new design (which I haven't figured out yet).

Which aspects do you want to modify ? @MasterOfTheTiger

**Comment by MasterOfTheTiger** [comment]
@locness3 The new header. For some reason my code wasn't running on the main page. It was probably a trivial issue though.

**Comment by moeenio** [comment]
> @locness3 The new header. For some reason my code wasn't running on the main page. It was probably a trivial issue though.

What do you want to modify in it ?

**Comment by MasterOfTheTiger** [comment]
@locness3 The whole header. Like you said on the other issue. I want to make the custom header provided by Reimagined Scratch apply to the front page, /messages, and /projects/xxxxxx.

**Comment by moeenio** [comment]
> @locness3 The whole header. Like you said on the other issue. I want to make the custom header provided by Reimagined Scratch apply to the front page, /messages, and /projects/xxxxxx.

Oh. I think it odesn't work since when you visit a new page, the server returns a blank page, which is populated by JS in front-end. And so, if your code starts too early, it doesn't work. Maybe there is a function called on full load in the Scratch code ?

**Comment by MasterOfTheTiger** [comment]
I'd have to get into this project again and start development before I can explain further. I don't know 100% what the issue was, but I believe I can get it working. It makes using Scratch much easier (especially with the discuss button)
